### PR TITLE
Update with getting started

### PR DESCRIPTION
**Pull request by maestersid**
Updating README to include an Introduction of the SDK and getting started guidelines.

### CODE REVIEW

**Review by evgeny-pol — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,62 @@
...
+
+Once you have created an app, you can obtain its **App Secret** on the **Getting Started** or **Manage App** sections of the App Center Portal.
+
+## 3. Add the App Center SDK to your project
+
+The App Center SDK is integrated by importing Unity Packages into your project.
+
+### 3.1 Download the package(s)
+
+The App Center Unity packages are downloaded from the [releases tab](https://github.com/Microsoft/AppCenter-SDK-Unity/releases). Download the package(s) that you want to use, from the latest release. Its name should be in the format **AppCenter{Analytics/Crashes/Distribute}-v{version}.unitypackage**.
```
Comment (patniko) [inline README.md:37]: Remove the last sentence.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,62 @@
...
+
+### 3.2 Import the package
+
+Now open your Unity project. Once it is open, simply open the Unity package you just downloaded. A pop-up window should appear in your Unity project containing a list of files. Select **Import**, and the SDK will be added to your project. Do this for each package you downloaded and plan to use with your project.
```
Comment (patniko) [inline README.md:41]: Don't say things like "Now", "Once", "simply". Be clear and direct.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,62 @@
...
+
+### 4.1 Create an empty Game Object
+
+App Center works as a component that you attach to a game object in the scene that your game launches into. Navigate to this scene and add an empty game object. Give it a descriptive name, such as "App Center".
+
+### 4.2 Attach the App Center script
+
+In the **Project** window, navigate to the "AppCenter" folder that was added to your project. Locate the script whose icon is the App Center logo, named *AppCenterBehavior*, and drag it onto your newly created game object in the **Hierarchy** window.
+
+> [!NOTE]
```
Comment (patniko) [inline README.md:53]: The !NOTE markdown isn't getting rendered correct. Would just remove that and let the right arrow control formatting for the main message.

File: README.md [hunk 4]
```diff
@@ -1,4 +1,62 @@
...
+You do not need to add App Center to every scene in which you wish to use it. Adding it to the first loaded scene is enough.
+
+### 4.3 Configure App Center settings
+
+Click on this new "App Center" object and add your app secrets to the corresponding fields in the **Inspector** window. Make sure to also check the "Use {service}" boxes for each App Center service you intend to use
```
Comment (patniko) [inline README.md:58]: Don't say things like "this". You aren't personally walking people through what they're doing so this needs to change.

**Review by patniko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**